extensions/amp-consent/0.1/consent-config.js

```diff
@@ -46,13 +48,16 @@ export class ConsentConfig {
 
   /**
    * Read validate and return the config
-   * @return {!JsonObject}
+   * @return {!Promise<JsonObject>}
    */
-  getConsentConfig() {
+  getConsentConfigPromise() {
```

Comment:
calling this methods multiple times might cause `this.validateAndParseConfig_()` being called multiple times.

to avoid that, you can change this.config_ to this.configPromise_

Reply:
Ahh, I see. Thanks